build.gradle

```diff
@@ -88,10 +88,10 @@ ospackage {
 buildRpm {
     arch = 'NOARCH'
     addParentDirs = false
-    archiveName "${packageName}-${version}.rpm"
+    archiveName "${packageName}-${project.version}.rpm"
```

Comment:
`version` for gradle is defined on line 59.

Reply:
@vrozov can you make the fix you recommend and merge it? I don't understand the specific feedback you're giving, and the overall PR is so small; I am feeling frustrated with all the rebasing, signing, two reviewers, etc.

Reply:
@cjcjameson It is not necessary to modify `build.gradle`. It is not part of the maven build and is not a subject of the maven deprecation warning that this PR is fixing.

Reply:
ok, gotcha. removed